plugins/inventory/terraform_state.py

```diff
@@ -13,9 +13,11 @@
   - This plugin works with an existing state file to create an inventory from resources created by cloud providers.
   - The plugin accepts a Terraform backend config to an existing state file or a path to an existing state file.
   - Uses a YAML configuration file that ends with terraform_state.(yml|yaml).
-  - To read the state file command ``Terraform show`` is used.
   - Does not support caching.
   - The Terraform providers for AWS, Azure and Google Cloud are supported by Red Hat Ansible. Other providers are supported by the community.
+notes:
+  - This plugin retrieves a local copy of the remote state file, and parses the raw state file.
+  - The current version of this plugin can parse the state file with version 4. 'version_added=3.1.0'
```

Comment:
I see version_added: 2.1.0 underneath, so, why 'version_added=3.1.0'?

Reply:
`2.1.0` is the initial release on which the plugin was added. This change is expected to be delivered into `3.1.0` release

Reply:
I don't think we need this note here. This is a bugfix PR and can be backported to the stable-2 branch.

Reply:
updated